app/assets/javascripts/app/models/post/interactions.js

```diff
@@ -44,12 +44,16 @@ app.models.Post.Interactions = Backbone.Model.extend({
   },
 
   userLike : function(){
-    return this.likes.select(function(like){ return like.get("author").guid === app.currentUser.get("guid")})[0];
+    return this.likes.select(function(like){
+      if(like.get("author") !== undefined) {
```

Comment:
Okay, so here is how I found this:
* hit 'like' for the first time on a post by someone who's ignoring you -> 422 error
* then hit the 'like' button again -> see what happens